tests/conftest.py

```diff
@@ -0,0 +1,158 @@
+from typing import Generator
```

Comment:
Please check with fern how tests can be run as part of CI

Reply:
The tests that I've written are ran by this action: https://github.com/humanloop/humanloop-python/actions/runs/11601419630/job/32304102387?pr=24

Do you mean something else?